packages/content-management/table_list/src/components/listing_limit_warning.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { EuiCallOut, EuiLink, EuiSpacer } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+interface Props<T> {
+  entityNamePlural: string;
+  canEditAdvancedSettings: boolean;
+  advancedSettingsLink: string;
+  totalItems: number;
+  listingLimit: number;
+}
+
+export function ListingLimitWarning<T>({
+  entityNamePlural,
+  totalItems,
+  listingLimit,
+  canEditAdvancedSettings,
+  advancedSettingsLink,
+}: Props<T>) {
+  return (
+    <>
+      <EuiCallOut
+        title={
+          <FormattedMessage
+            id="contentManagement.tableList.listing.listingLimitExceededTitle"
+            defaultMessage="Listing limit exceeded"
+          />
+        }
+        color="warning"
+        iconType="help"
+      >
+        <p>
+          {canEditAdvancedSettings ? (
+            <FormattedMessage
+              id="contentManagement.tableList.listing.listingLimitExceededDescription"
+              defaultMessage="You have {totalItems} {entityNamePlural}, but your {listingLimitText} setting prevents
```

Comment:
This message is awfully similar to the other one, except for the last sentence. Can they somehow be one message?

Reply:
Fixed in https://github.com/elastic/kibana/pull/135892/commits/10a7dbb40c9988e34b3a3f8f2b1512630f2f42bf